config/locales/en.yml

```diff
@@ -267,38 +267,54 @@ en:
       search_for_a_service_button: Search
   homepage:
     categories:
-      abroad: Passports, travel and living abroad
-      abroad_description: Includes renewing passports and travel advice by country
-      benefits: Benefits
-      benefits_description: Includes eligibility, appeals, tax credits and Universal Credit
-      births_deaths_marriages: Births, deaths, marriages and care
-      births_deaths_marriages_description: Parenting, civil partnerships, divorce and Lasting Power of Attorney
-      business: Business and self-employed
-      business_description: Tools and guidance for businesses
-      childcare_parenting: Childcare and parenting
-      childcare_parenting_description: Includes giving birth, fostering, adopting, benefits for children, childcare and schools
-      citizenship: Citizenship and living in the UK
-      citizenship_description: Voting, community participation, life in the UK, international projects
-      disabilities: Disabled people
-      disabilities_description: Includes carers, your rights, benefits and the Equality Act
-      driving: Driving and transport
-      driving_description: Includes vehicle tax, MOT and driving licences
-      education: Education and learning
-      education_description: Includes student loans, admissions and apprenticeships
-      employing_people: Employing people
-      employing_people_description: Includes pay, contracts, hiring and redundancies
-      environment_countryside: Environment and countryside
-      environment_countryside_description: Includes flooding, recycling and wildlife
-      housing_local_services: Housing and local services
-      housing_local_services_description: Owning or renting and council services
-      justice: Crime, justice and the law
-      justice_description: Legal processes, courts and the police
-      tax: Money and tax
-      tax_description: Includes debt and Self Assessment
-      visas_immigration: Visas and immigration
-      visas_immigration_description: Apply to visit, work, study, settle or seek asylum in the UK
-      working: Working, jobs and pensions
-      working_description: Includes holidays, finding a job and redundancy
+      - title: Benefits
```

Comment:
Could we add a comment here reminding folks to update columns like we have for the super nav?

Reply:
Good idea - done in https://github.com/alphagov/frontend/pull/3033/commits/3de50eb9bf9dbe3add9f16ed084eebec5f87d7fa.